eval/eval.py

```diff
@@ -608,9 +1201,85 @@ def output_token_scores() -> None:
             write_basic_value(wrong_span, "wrong_span", num_tokens, tp)
             calc_recall(right_span, wrong_span, "right_span", "wrong_span", "spanrecall")
 
-        output_duration()
-        output_sentence_scores()
-        output_token_scores()
+        def output_error_cat_scores() -> None:
+            """ Calculate and write scores for each error category to stdout"""
+            print("\n\nResults for each error category in order by frequency")
```

Comment:
I'm a bit worried about the `print()` statements. There was some careful coding in `eval.py` to collect output separately and print it in correct order even with multiple processes running in parallel.

Reply:
Changed to bprint.